pwa/src/components/attributes/attribuutForm.tsx

```diff
@@ -144,25 +156,22 @@ export const AttributeForm: React.FC<AttributeFormProps> = ({ attributeId, entit
 
     if (!attributeId) { // unset id means we're creating a new entry
       API.Attribute.create(body)
-        .then((res) => {
-          setAttribute(res.data)
-          navigate(`/entities/${entityId}`)
-        })
+      navigate(`/entities/${entityId}`)
         .catch((err) => {
-          setAlert({ type: 'danger', message: err.message });
-          throw new Error ('Create application error: ' + err)
+          setAlert({type: 'danger', message: err.message});
+          throw new Error('Create application error: ' + err)
         })
     }
 
     if (attributeId) { // set id means we're updating a existing entry
       API.Attribute.update(body, attributeId)
         .then((res) => {
-          setAttribute(res.data)
-          navigate(`/entities/${entityId}`)
+          setAttribute(res.data);
+          setShowSpinner(false);
```

Comment:
The `setShowSpinner(false)` should not happen in the `.then()`, but rather in the `.finally()`.

Reasoning behind that is: when there's an error, the `Spinner` will always stay, but logically it should be removed when the request is finished regardless of the outcome.

Reply:
Please check for all files changed in this pull request